docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,107 @@
+
+To get the correct UUID of your instance you can use the following command:
+
+```bash
+openstack server list
+```
+
+## Creating a snapshot
+
+To start the snapshot process you use the following command:
+
+```bash
+openstack server image create --wait --name <new-image-name> <uuid-of-instance>
+```
+
+After a while you will get some output showing information of the new image, such as the current state and the image UUID.
+
+Before going on the state needs to be active and you can see the current state with the following command:
+
+```bash
+openstack image show <uuid>
+```
+
+## Download your snapshot
+
+Before you can download the image you need to know what format the image has, to find this out you can run the following command:
+
+```bash
+openstack image show <image_id> | grep disk_format
+```
+
+To download the image to the local machine you use the following command:
+
+```bash
+openstack image save --file <local image name>.<disk_format> <your_image_id>
+```
+
+When the file have been downloaded, you are done with the steps on the source side, and the following steps will be done on the target side.
+
+## Target Location
+
+Load the rc-file for the target side. Instructions how to generate this can be found in the link to OpenStack client installation in the beginning of this guide.
+
+```bash
+source <target_openrc.sh>
+```
+
+## Upload your snapshot
+
+Start upload the file to the new location and use disk_format from above and also the name you want to give the image.
+
+```bash
+openstack image create --disk-format <disk_format> --file snapshot.<disk_format> <new_image_name>
+```
+
+Upload will take a while, and after getting your prompt back, you can check the status by using the following command.
+
+```bash
+openstack image list | grep <new_image_name>
+```
+
+When status is active, the upload is done.
+
+> After you have moved your instance and don't want to keep the old one, don't forget to remove it to save some cost.
+
+## Create a new server
+
+Next step you need to do, is to create a new server from the uploaded image. This can be done either with CLI commands or via the [Control panel](https://cleura.cloud).
+
+To create a new server from the control panel you do the following:
+
+Login to your account and make sure the region you uploaded to are being used.
+
+In the left side panel choose Images and Private Images and you will see the uploaded image.
+
+Click on the orange 3 dots to the right of the disk format of your image and choose to create a new server.
+
+Provide the new server with a name, boot target (boot from volume is recommended), flavor and size of disk. Choose which network and security group that should be used.
+
+> The volume needs to be at least the same size as the source server.
+
+If you want to use password to your server you need to add the below cloud-config under Advanced Options and User-Data
+
+```bash
+# cloud-config
+password: mypasswd
+chpasswd: { expire: False }
+ssh_pwauth: True
+```
```

Comment:
I think once #56 lands, we can remove this whole section with a reference to that how-to.

Reply:
I read on the #56 but I could not find that there was a reference to how to create a server from an image. Should we still drop this part and replace it with #56 ?

Reply:
https://docs.cleura.cloud/howto/openstack/nova/new-server/#creating-a-server — recommends to boot a server off of an Ubuntu 22.04 image. So, yes. Let's please just replace this with a cross-reference.